google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/CloseStream.java

```diff
@@ -46,6 +49,13 @@ public static CloseStream fromProto(@Nonnull ReadChangeStreamResponse.CloseStrea
         closeStream.getStatus(),
         closeStream.getContinuationTokensList().stream()
             .map(ChangeStreamContinuationToken::fromProto)
+            .collect(ImmutableList.toImmutableList()),
+        closeStream.getNewPartitionsList().stream()
```

Comment:
do we need to check if status is OK then new partition should be empty?  what will happen in this case?

Reply:
The API should guarantee that only one of the following can happen:
1) Status==OK, new_partitions==empty, continuation_tokens==empty
2) Status!=OK, new_partitions!=empty, continuation_tokens!=empty, in this case new_partitions.size() == continuation_tokens.size()

Added some Preconditions.checkState